api/nakadi-event-bus-api.yaml

```diff
@@ -351,6 +353,54 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+    delete:
+      tags:
+        - schema-registry-api
+      security:
+        - oauth2: [nakadi.admin]
+      description: |
+        Deletes an `EventType` identified by its name. The underlying Kafka topic and all events of this `EventType`
```

Comment:
I my eyes, they shouldn't. We may change this at any point in time. It might help people who are familiar with Kafka to understand some of the Nakadi principles, but in general Nakadi is not about Kafka.

The thing they should be aware of in terms of deletion is that they delete the topic including all events permanently. That physically a Kafka topic is deleted is implementation details.